lib/src/local/request.rs

```diff
@@ -189,6 +189,29 @@ impl<'c> LocalRequest<'c> {
         self
     }
 
+    /// Add a [private cookie] to this request.
+    ///
+    /// [private cookie]: /rocket/http/enum.Cookies.html#private-cookies
+    ///
+    /// # Examples
+    ///
+    /// Add `user_id` as a private cookie:
```

Comment:
This says `user_id` but a "username" cookie was added in the example.

Reply:
ooh, silly mistake.